controller/scheduler/swarm.py

```diff
@@ -48,14 +50,20 @@ def stop(self, name):
         """
         Stop a container
         """
-        self.docker_cli.stop(name)
+        try:
+            self.docker_cli.stop(name)
+        except errors.APIError:
+            raise errors.DockerException("cannot stop container " + name)
```

Comment:
I think we still need to capture these in models.py so we don't return a 500 response with no error message, if I'm not mistaken. I'll build this PR maually and test this out.